static/flow/views/pi-selector-panel.js

```diff
@@ -139,7 +147,9 @@ var PiSelectorPanel = function(options) {
                         }
                         piMenuData[controller.name] = controller.name;
                     }
- 
+//piMenuData["RPi01"] = "RPi01";
+//piMenuData["RPi02"] = "RPi02";
+//piMenuData["RPi03"] = "RPi03";
```

Comment:
Dead code

Reply:
Yup.  That was a test case that got left in!  Good catch!